### DISCUSSION TITLE
Step 4 issues

### DISCUSSION THREAD

**Discussion by Descent098** [Descent098/ezcv#16] [Issues]
This thread is for people who are having issues up to step 4 on the [Quick Start](https://ezcv.readthedocs.io/en/latest/quick-start/#remote-editing). Please post your site repo URL below i.e. ```https://github.com/<username>/<username>.github.io```

**Comment by debMan** [comment]
Some problem, No background image:
https://github.com/debMan/debMan.github.io

**Reply by Descent098** [reply]
> Some problem, No background image: https://github.com/debMan/debMan.github.io

So it looks like github pages is reading your main branch instead of the gh-pages branch. Go into _Settings --> Pages --> Source_ and set the source to the gh-pages branch. Let me know if that fixes it for you, Thanks!

![image](https://user-images.githubusercontent.com/22761705/138383292-0b91e863-ec4e-4193-a244-fa76af485804.png)

**Reply by debMan** [reply]
Worked like a charm. Thanks. :+1:

**Reply by Descent098** [reply]
No problem, feel free to post if you run into other issues 😄

**Reply by devonanirudh** [reply]
Having a problem where the build passes but it displays configuration values instead of a rendered page: https://github.com/devonanirudh/devonanirudh.github.io

**Reply by Descent098** [reply]
@devonanirudh Looks like you have your theme set to "base", this theme is a theme for developers to help them see how state works in ezcv. You can see this line set here in your `config.yml` https://github.com/devonanirudh/devonanirudh.github.io/blob/main/config.yml#L24

You can change that to another theme any you'll be good to go. [This is the list of themes](https://ezcv.readthedocs.io/en/latest/included-themes/), and for each there is [a section](https://ezcv.readthedocs.io/en/latest/included-themes/#dimension:~:text=Other%20configuration%20settings-,theme%3A%20dimension,-...%20%23%20Other%20configuration) where you can see their name for the config.yml file.

I recommend dimension because it has every feature implemented!

For that change `base` to `dimension` in `config.yml` and you're good to go!

Let me know if you run into any issues!
